src/components/search/data/tests/searchLoader.test.jsx

```diff
@@ -0,0 +1,121 @@
+import { screen, waitFor } from '@testing-library/react';
```

Comment:
[inform] A `searchLoader.test.jsx` test suite is already defined on `master` (recently added); not showing up as merge conflict(s) due to the different file location of your test suite here, though.